content/_releases/22/36.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Product Update - v22.36
+description: Details of the product update v22.36 from 8th of September 2022.
+layout: rel
+version: "22.36"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-09-08
+redirect_from:
+  - /releases/22.36.html
+---
+
+## Improvements and Updates
+
+### Credentials page: new design
+
+This is continuation of our effort to modernise and harmonise look and feel of the
```

Comment:
**the** look and feel

Reply:
We continue to update our User Interface to improve ease of use and reduce UI lag. This time it is the UI surrounding Credentials that is updated. When you navigate to Integrate > Credentials you will notice larger icons and a more coherent design.